frontend/packages/component-library/src/cms/iconHighlight/IconHighlight.tsx

```diff
@@ -0,0 +1,97 @@
+import classNames from 'classnames';
+import React, {Key, ReactNode, HTMLAttributes} from 'react';
+
+import FontAwesomeV6Icon, {FontAwesomeV6IconProps} from '@/fontAwesomeV6Icon';
+import Link, {LinkProps} from '@/link';
+import {Heading5, BodyThreeText} from '@/typography';
+
+import moduleStyles from './iconHighlight.module.scss';
+
+export const ICON_HIGHLIGHT_DEFAULT_ICON = 'smile';
+
+export type IconHighlightLinkProps = LinkProps & {
+  key: Key;
+  external?: boolean;
+};
+
+export interface IconHighlightProps extends HTMLAttributes<HTMLElement> {
+  /** IconHighlight heading */
+  heading: string | ReactNode;
+  /** IconHighlight content */
+  text: string | ReactNode;
+  /** IconHighlight icon */
+  icon?: FontAwesomeV6IconProps;
+  /** IconHighlight links */
+  links?: IconHighlightLinkProps[];
+  /** IconHighlight class  */
+  className?: string;
+}
+
+/**
+ * ## Production-ready Checklist:
+ *  * (✔) implementation of component approved by design team;
+ *  * (✔) has storybook, covered with stories and documentation;
+ *  * (✔) has tests: test every prop, every state and every interaction that's js related;
+ *  * (see ./__tests__/IconHighlight.test.tsx)
+ *  * (✔) passes accessibility checks;
+ *
+ * ### Status: ```Ready for dev```
+ *
+ * Design System: IconHighlight Component.
+ * Renders a card with a customizable icon, heading, text, and optional links.
+ */
+const IconHighlight: React.FC<IconHighlightProps> = ({
+  heading,
+  text,
+  className,
+  icon = {
+    iconName: ICON_HIGHLIGHT_DEFAULT_ICON,
+    iconStyle: 'solid',
+  },
+  links = [],
+  ...HTMLAttributes
+}: IconHighlightProps) => (
+  <aside
+    {...HTMLAttributes}
+    className={classNames(moduleStyles.iconHighlight, className)}
+  >
+    <FontAwesomeV6Icon
+      {...icon}
+      role="presentation"
+      aria-hidden="true"
+      className={classNames(moduleStyles.iconHighlightIcon, icon.className)}
+    />
+
+    <Heading5 className={moduleStyles.iconHighlightHeading}>{heading}</Heading5>
```

Comment:
For better accessibility let's make this an `h3` since that's the hierarchy it'll be on the page:
```suggestion
    <Heading3 visualAppearance={'heading-sm'} className={moduleStyles.iconHighlightHeading}>{heading}</Heading5>
```